### ISSUE TITLE
Unusable as is

### ISSUE THREAD

**Issue by yo6nam** [wojciechk8/MMDVM_pog#6]
For a hobbyist this project is unusable. There are so many things and steps you need to make, apart from (installing VM, linux OS, readme steps), just to be able to export a copy of gerber files and BOM. I'm in the second day and still struggling with the current state of the files to make something that works or at least exports.
"gsch2pcb mmdvm_pog.prj" produces an error because there is no such thing as "partslist4"

Although I appreciate the author's efforts and the fact that it's been public since 2017, but it's funny to see this open source adapted (by some hams, china factories), modified and transformed into a commercial product and still, someone trying to get a PCB out of this hits so many problems.
If someone has knowledge with gEDA, please step in and make a few mods to this project :

- add a voltage divider (100K pot) for J21 pin 8 to STM32F105RBT pin 27
- add 2 connections from J3 pin 38 to STM32F105RBT pin 7 (NRST) and J3 pin 40 to STM32F105RBT pin 60 (BOOT0) to avoid this hack :
https://github.com/yo6nam/MMDVM_STMF1XX_FW/blob/master/pog_stm_update_mod.jpg
- (optional) P31 and P41 footprint changed. Parts will be replaced with :
https://www.tme.eu/en/details/t67w-10k/1-4-inch-multiturn-tht-trimmers/sr-passives/

**Comment by N4IRS** [comment]
One thing you may want to consider is that STM32F105 is not fast enough to support all modes. It will do D-Star, DMR, YSF, P25 but not NXDN.

**Comment by yo6nam** [comment]
Hi Steve,

We're not using NXDN in my area so there's not much to loose. DMR and C4FM are the only modes that people in YO are using. I just want to build an improved version of this board and keep the costs low. Now that JLCPCB launched the SMT Assembly I could build a board for about 15 USD / pcs. 

Regards and '73
Razvan / YO6NAM

**Comment by k7ilo** [comment]
Yeah I know this is a old thread but I just wanted to know if there were any updates to this board or even the schematic files before i copy this schematic into later software and create newer gerber files.  JLCPCB will not except, or i think they wont except these versions of gerber.
Lucily, the parts list is here so i can make sure parts that were used for this design are still available.

But again, are there any updates around for this design?

Thanks

Harold
K7ILO
